### PR TITLE
API route clarification

### PR DESCRIPTION
**Pull request by Jutanium**
- Specify what an API route is
- Add the example filename to match the previous example
- Add `async` in the function declaration so that it's a working example

### CODE REVIEW

**Comment by sarah11918** [comment]
@matthewp Can you please verify that this change is good by you? If you're happy with it, feel free to merge.

File: src/pages/en/guides/server-side-rendering.md [hunk 1]
```diff
@@ -119,12 +119,13 @@ if(!product) {
 
 #### API Routes
 
-A [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) can also be returned from an API route.
+An API route is a .js or .ts file in the src/pages/ folder that takes a request and returns a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response).
```
Comment (aFuzzyBear) [inline src/pages/en/guides/server-side-rendering.md:122]:
```suggestion
An API route is either a `.js` or `.ts` file within the `src/pages/` folder which  takes a [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request) and returns a `[Response]`(https://developer.mozilla.org/en-US/docs/Web/API/Response).
```

Comment (aFuzzyBear) [reply src/pages/en/guides/server-side-rendering.md:122]: Just a small formatting nit

**Review by aFuzzyBear — COMMENTED** [review]
Aye, lets wait for the big yin to sign off on this =), looks good @Jutanium

File: src/pages/en/guides/server-side-rendering.md (hunk 1 above)
Comment (sarah11918) [reply src/pages/en/guides/server-side-rendering.md:122]:
```suggestion
An API route is a `.js` or `.ts` file within the `src/pages/` folder that takes a [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request) and returns a `[Response]`(https://developer.mozilla.org/en-US/docs/Web/API/Response).
```

Aaand.... this is what I would do! (I don't think "either" needs to be there, and I feel like "that" is more in the voice of these docs than "which.")

**Comment by Jutanium** [comment]
I can't imagine @matthewp would have a problem with this seeing that I took his language from his blog post, and since the code is broken I think we should merge it and change later if there's a problem.

File: src/pages/en/guides/server-side-rendering.md (hunk 1 above)
Comment (aFuzzyBear) [reply src/pages/en/guides/server-side-rendering.md:122]: English is a romantic language

“Speak the speech, I pray you, as I pronounc'd it to you, trippingly on the tongue.”

**Comment by sarah11918** [comment]
OK, then I'm commiting the code formatting and extra link from Fuzzy, and I'll take the blame for this one...

File: src/pages/en/guides/server-side-rendering.md [hunk 2]
```diff
@@ -119,12 +119,13 @@ if(!product) {
 
 #### API Routes
 
-A [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) can also be returned from an API route.
+An API route is a `.js` or `.ts` file within the `src/pages/` folder that takes a [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request) and returns a `[Response]`(https://developer.mozilla.org/en-US/docs/Web/API/Response).
```
Comment (sarah11918) [inline src/pages/en/guides/server-side-rendering.md:122]:
```suggestion
An API route is a `.js` or `.ts` file within the `src/pages/` folder that takes a [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) and returns a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response).
```

**Pull request merged**